composer.json

```diff
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
+        "php": "^8.0",
```

Comment:
This seems contradictory, why would enforcing PHP 8 be desired if it works for PHP 5.6+?

Also why keep test matrix for PHP <8 if we're saying support is only for PHP 8?

Reply:
Thanks for the feedback. I was looking at upgrading this module to support php8, haven't fully finished it. Sorry, will close it for now